modules/drivers/druid/src/metabase/driver/druid/client.clj

```diff
@@ -86,7 +86,7 @@
           ;; Re-throw a new exception with `message` set to the extracted message
           (throw e'))))))
 
-(defn- cancel-query-with-id! [details query-id]
+(defn- undo-query-with-id! [details query-id]
```

Comment:
I find this name to be more accurate since a) we already have cancellation; b) this is indeed more than just simple cancellation as it goes into the DB to do explicit deletion.